schemas/pulse-job.yml

```diff
@@ -116,9 +124,11 @@ properties:
       - unknown
   jobKind:
     type: "string"
+    default: "other"
     enum:
       - build
       - test
+      - other
   tier:
```

Comment:
I'm okay with this... But warning you that when designing a  schema there is little "for now" :)
If you have 10 services feed into this, "for now" becomes "forever" very easily... Doesn't mean you can't change, just means you'll likely have to convert old value to new ones in your ingestion code.

Granted you can always change to `enum: [1,2,3 ,2.5]` that'll be backwards compatible. So this is fine.